src/openapi-generator.ts

```diff
@@ -710,6 +710,15 @@ export class OpenAPIGenerator {
       return {};
     }
 
+    if (isZodType(zodSchema, 'ZodDate')) {
+      const metadata = this.getMetadata(zodSchema);
+      return {
+        type: 'string',
+        nullable: isNullable ? true : undefined,
+        example: metadata?.example?.toISOString(),
```

Comment:
Seeing that you are going with the `toISOString` approach then we could have added a `date-time` `format`. However, looking at [this](https://swagger.io/docs/specification/data-models/data-types/) I see that there are both `date` and `date-time`. This means that for a "user" of `zod-to-openapi` it might be a perfectly valid use-case to use a `date` format. And since the examples are being set by the "user" I'd rather leave this to them to serialize and not touch it at all here.